### ISSUE TITLE
"My nation is great" Too vague to be useful measure.

### ISSUE THREAD

**Issue by spelletier1996** [8values/8values.github.io#96]
The question "my nation is great" is far too vague to offer any meaning full insight. 

This can be interpreted from either a nationalistic view or a pragmatic view. I could argue that what my country is great compared to a country that doesn't align as well with my view.

Or another interpretation would be that no matter the situation my nation is great which would be a nationalistic take. 

A better question if its being used to determine nationalistic view could be "my country is great no matter what it does" for example.

**Comment by SouthernWolf95** [comment]
I can see why the question was originally included. However, I also agree that perhaps this needs to be expanded upon slightly to differentiate if from being either an explicitly nationalistic question or one based on a more philosophical question. The first could be what you said "my country is great no matter what it does," while the second one could be "My country is the great because of it's moral values," or something of this nature.

**Comment by Genora51** [comment]
I'm pretty sure this is a question of nationalism rather than a specific view of its values. So probably the former would be a better change, although I think that question would perhaps be too strongly worded (some people who would align more to the nationalistic side might still disagree with the sentiment).

So if this could be reworded further from "My country is great no matter what it does", just to tone it down a little so that it's an even Agree/Disagree split then that would be a big improvement to a question that is currently a bit vague.

**Comment by DefunctLizard** [comment]
change it to " my nation is the best one "  ?                something like that......

**Comment by nissubjtsdjarwylwz** [comment]
Yes, the question is completely retarded. I don't think my nation is great, nor I believe my nation is something more than a legal construct.  But I do put it's interests over others and it is in my interests to make it great.  Simply pragmatic approach.

**Comment by Genora51** [comment]
Please be civil in the discussion; these pages are not intended as an assessment of whether or not the views expressed in the question are right. It doesn't matter whether you agree with the sentiment, because that's what the quiz is asking. These discussions should instead be focused around whether the questions are a useful measure of one of the axes of the test.

Also, please refrain from using the slur, it's not productive or helpful.

**Comment by nissubjtsdjarwylwz** [comment]
> Also, please refrain from using the slur, it's not productive or helpful.

Not everything should be productive and helpful. Some things just can exist, Like your comment for example. What is not acceptable is censorship.

**Comment by DefunctLizard** [comment]
> Yes, the question is completely retarded. I don't think my nation is currently great and we have a lot of work to do, nor I believe any nation is something more than an abstract legal construct that exists only on paper, we have a lot of ethnicities here but we all work on a common goal.  But I do put it's interests over others and it is in my interests to make it great.  Simply pragmatic approach.

So.... your answer to the question on the test would be no. But there are other people that would answer it yes. The point of this discussion is how useful the for the test would it be to distinguish between te people that answer yes and the people that answer no.

Tldr: Your response is not related to the topic.

**Comment by phoebe-leong** [comment]
It would greatly improve the question to change it to "My nation is the greatest one." or something like that. 
If people misinterpret the question then they might get the wrong score.

**Comment by womanburner** [comment]
> > Also, please refrain from using the slur, it's not productive or helpful.
> 
> Not everything should be productive and helpful. Some things just can exist, Like your comment for example. What is not acceptable is censorship.

based and redpilled
